pkg/apis/networking/validation/validation_test.go

```diff
@@ -246,6 +276,38 @@ func TestValidateNetworkPolicy(t *testing.T) {
 						},
 					},
 				},
+				Egress: []networking.NetworkPolicyEgressRule{
+					{
+						To: []networking.NetworkPolicyPeer{
+							{
+								PodSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{"c": "d"},
+								},
+								NamespaceSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{"c": "d"},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		"no from and to type": {
```

Comment:
Can you change this to 

`missing from and to type`

Reply:
Thanks! Done.